src/graphql/types/abstract/initiation-via.ts

```diff
@@ -0,0 +1,48 @@
+import { GT } from "@graphql/index"
+
+import WalletId from "../scalar/wallet-id"
+import Username from "../scalar/username"
+import OnChainAddress from "../scalar/on-chain-address"
+import { PaymentInitiationMethod } from "@domain/wallets"
+import PaymentHash from "../scalar/payment-hash"
+
+const InitiationViaIntraLedger = new GT.Object({
+  name: "InitiationViaIntraLedger",
+  isTypeOf: (source) => source.type === PaymentInitiationMethod.IntraLedger,
+  fields: () => ({
+    counterPartyWalletId: {
+      // type: GT.NonNull(WalletId),
```

Comment:
Based on https://github.com/GaloyMoney/galoy/pull/771#issuecomment-983855316 should I change this back as well?